l1-contracts/contracts/state-transition/DualVerifier.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {IVerifier} from "./chain-interfaces/IVerifier.sol";
+import {ZKChainStorage} from "./chain-deps/ZKChainStorage.sol";
+import {ZeroAddress} from "../common/L1ContractErrors.sol";
+
+/// @title Dual Verifier
+/// @author Matter Labs
+/// @notice Wrapper contract to verify a zk-SNARK proof based on the proof type
+/// @custom:security-contact security@matterlabs.dev
+contract DualVerifier is IVerifier{
+
+    // slither-disable-next-line uninitialized-state
+    ZKChainStorage internal s;
```

Comment:
Verifier is a stand alone application, should have its own storage. But actually it can have no storage and works fine